packages/webb-ui-components/src/provider/index.tsx

```diff
@@ -24,5 +20,14 @@ export const WebbUIProvider: React.FC<WebbUIProviderProps> = ({ children }) => {
     [isDarkMode, toggleMode]
   );
 
-  return <WebbUIContext.Provider value={{ theme }}>{children}</WebbUIContext.Provider>;
+  return (
+    <WebbUIContext.Provider value={{ theme }}>
+      {hasErrorBoudary ? (
+        // @ts-ignore
```

Comment:
The `@ts-ignore` here for overcome the `ts-check` command. Because it is related to type definition which is not compatible with `React`. We have used the [same approach](https://github.com/webb-tools/webb-dapp/blob/develop/packages/apps/src/App.tsx#L18-L19) in the dApp.

<img width="1440" alt="CleanShot 2022-09-20 at 18 19 05@2x" src="https://user-images.githubusercontent.com/60747384/191244623-422c65a7-13f1-4044-b87d-56491099dcc5.png">

Reply:
@AtelyPham  Use the  `React.createElement`

``` tsx
const WebbUIEErrorBoundaryElement = useMemo(() => {
  return React.createElement(WebbUIErrorBoundery, {
    logger:appLogger
    children,
  });
}, [appLogger, children])

  ```